golang/cosmos/Makefile

```
@@ -147,66 +166,15 @@ proto-update-deps:
 	mkdir -p $(SDK_QUERY_TYPES) && \
 	curl -sSL $$url/base/query/v1beta1/pagination.proto > $(SDK_QUERY_TYPES)/pagination.proto && \
 	mkdir -p $(SDK_UPGRADE_TYPES) && \
-	curl -sSL $$url/upgrade/v1beta1/upgrade.proto > $(SDK_UPGRADE_TYPES)/upgrade.proto
-
-UNAME_S ?= $(shell uname -s)
-UNAME_M ?= $(shell uname -m)
-
-BUF_VERSION ?= 1.0.0
-
-PROTOC_VERSION ?= 3.11.2
-ifeq ($(UNAME_S),Linux)
-  ifeq ($(UNAME_M),aarch64)
-    PROTOC_ZIP ?= protoc-${PROTOC_VERSION}-linux-aarch_64.zip
-  else
-    PROTOC_ZIP ?= protoc-${PROTOC_VERSION}-linux-x86_64.zip
-  endif
-endif
-ifeq ($(UNAME_S),Darwin)
-  PROTOC_ZIP ?= protoc-${PROTOC_VERSION}-osx-x86_64.zip
-endif
-
-proto-tools: proto-tools-stamp buf grpc-gateway-stamp
-
-proto-tools-stamp:
-	echo "Installing protoc compiler..."
-	mkdir -p "${BIN}"
-	(cd /tmp; \
-	curl -OL "https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/${PROTOC_ZIP}"; \
-	unzip -o ${PROTOC_ZIP} -d "${BIN}/.." bin/protoc 'include/*'; \
-	rm -f ${PROTOC_ZIP})
-
-	echo "Installing protoc-gen-gocosmos..."
-	go install github.com/regen-network/cosmos-proto/protoc-gen-gocosmos
-
-	# Create dummy file to satisfy dependency and avoid
-	# rebuilding when this Makefile target is hit twice
-	# in a row
-	touch $@
-
-buf: buf-stamp
-
-buf-stamp:
-	echo "Installing buf..."
-	curl -sSL \
-    "https://github.com/bufbuild/buf/releases/download/v${BUF_VERSION}/buf-${UNAME_S}-${UNAME_M}" \
-    -o "${BIN}/buf" && \
-	chmod +x "${BIN}/buf"
-
-	touch $@
-
-GATEWAY_PKGS = \
-	github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway \
-	github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger \
-	google.golang.org/protobuf/cmd/protoc-gen-go
-
-grpc-gateway-stamp:
-	go install $(GATEWAY_PKGS)
-	touch $@
+	curl -sSL $$url/upgrade/v1beta1/upgrade.proto > $(SDK_UPGRADE_TYPES)/upgrade.proto && \
+	mkdir -p $(SDK_MSG_TYPES) && \
+	curl -sSL $$url/msg/v1/msg.proto > $(SDK_MSG_TYPES)/msg.proto
 
-tools-clean:
-	rm -f proto-tools-stamp buf-stamp grpc-gateway-stamp
+	url="$(COSMOS_SDK_AMINO_URL)"; \
+	mkdir -p $(COSMOS_SDK_AMINO_TYPES) && \
+	curl -sSL $$url/amino.proto > $(COSMOS_SDK_AMINO_TYPES)/amino.proto
 
+.PHONY: proto-all proto-gen proto-swagger-gen proto-format proto-lint proto-check-breaking proto-update-deps
```

Some of these targets appear not to exist.
```suggestion
.PHONY: proto-all proto-lint proto-gen proto-check-breaking proto-update-deps
```

---

:+1: